src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1652,6 +1664,12 @@ public Section updateSection(Section section) {
                 JSON, json(Collections.singletonMap("section", section))), handle(Section.class, "section")));
     }
 
+    public Translation createSectionTranslation(Long sectionId, Translation translation) {
+        checkHasSectionId(sectionId);
```

Comment:
Why do we validate the Id of the section while in updateSectionTranslation we are verifying the Id of translation ?